Circles fail

So, basically circles are not working for me :)
```
 Error: Invalid value for <circle> attribute transform="translate(NaN,NaN)"
d3.v3.min.js:1 Error: Invalid value for <text> attribute transform="translate(NaN,NaN)"
d3.v3.min.js:1 Error: Invalid value for <circle> attribute r="NaN"
```
this is a new feature, I even don't know what exatly it is for

--- a/packages/core/circles/server/models/circle.js
+++ b/packages/core/circles/server/models/circle.js
@@ -79,21 +79,23 @@ CircleSchema.statics.buildPermissions = function(callback) {
 
 
 var buildTrees = CircleSchema.statics.buildTrees = function(data) {
-  var tree = []
+  var tree = [];
 
   for (var index in data) {
     buildTree(data, index, tree);
   }
 
-  return tree;
+  return {
+    'name': 'circles',
+    'children': tree
+  };
 };
 
 function buildTree(data, id, branch) {
-
   var length = branch.length;
 
   branch.push({
-    "name": data[id].name
+    'name': data[id].name
   });
 
   if (hasChildren(data, id)) {
@@ -129,4 +131,5 @@ function noChildren(data, id) {
 function hasChildren(data, id) {
   return !noChildren(data, id);
 }
+
 mongoose.model('Circle', CircleSchema);

